### PR TITLE
✨ Feature/mb add ecdna to prod

### PR DESCRIPTION
**Pull request by migbro**
## Description

This PR adds the previously reviewed standalone workflow of AmpliconArchitect (AA) to the main production workflow. The main README has been updated to reflect the addition of this workflow as well as some minor cleanup to the standalone. The committed workflow only really differs in documentation, and trimming of a couple outputs from what was tested. Given that a full run is about $30, unless deemed necessary, I will not re-test.

Fixes # https://github.com/d3b-center/bixu-tracker/issues/1516

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Skip AA if WXS: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/782f792e-c25f-4af5-afbb-767cc8b13ab1
- [ ] Fully incorporated WGS: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/f71a35db-985c-4b86-b8f5-077cfd210df8

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,4 +253,3 @@ You can use the `include_expression` `Filter="PASS"` to achieve this.
 ## Other Resources
 - dockerfiles: https://github.com/d3b-center/bixtools
 
-![pipeline flowchart](./docs/kfdrc-somatic-variant-workflow.cwl.png)
```
Comment (dmiller15) [inline README.md:243]: Perhaps we should delete these PNGs from the repo as well

Comment (dmiller15) [reply README.md:243]: Also this leaves a trailing empty line. That's why you end up with a `|+` in the doc.

File: workflow/kfdrc-somatic-variant-workflow.cwl [hunk 2]
```diff
@@ -44,24 +45,24 @@ doc: |
...
-  | [kfdrc_production_strekla2_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_strekla2_wf.cwl)         |     |  x  |    |
-  | [kfdrc_production_theta2_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_theta2_wf.cwl)             |     |     |  x |
-  | [kfdrc_production_cnvkit_theta2_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_cnvkit_theta2_wf.cwl)|  x |     |    |
-  | [kfdrc_production_vardict_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_vardict_wf.cwl)           |     |  x  |    |
-  | [kfdrc_gatk_cnv_somatic_pair_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/sub_workflows/kfdrc_gatk_cnv_somatic_pair_wf.cwl)|  x  |     |    |
-
+  | Workflow                                                                                                                                            | CNV | SNV | SV | ecDNA |
```
Comment (dmiller15) [inline workflow/kfdrc-somatic-variant-workflow.cwl:48]: If feels weird to include ecDNA in here. I feel like the output from the ecDNA stuff isn't really variant information. It's more like stats.

File: workflow/kfdrc-somatic-variant-workflow.cwl [hunk 3]
```diff
@@ -44,24 +45,24 @@ doc: |
...
+  | [kfdrc_production_vardict_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_vardict_wf.cwl)           |     |  x  |    |       |
+  | [kfdrc_gatk_cnv_somatic_pair_wf.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/sub_workflows/kfdrc_gatk_cnv_somatic_pair_wf.cwl)|  x  |     |    |       |
+  | [kfdrc_production_amplicon_architect.cwl](https://github.com/kids-first/kf-somatic-workflow/blob/master/workflow/kfdrc_production_amplicon_architect.cwl)|     |     |      |   x  |
   #### SNV Callers
 
   - [Strelka2](https://github.com/Illumina/strelka/tree/v2.9.3) `v2.9.3`, from Illumina, calls single nucleotide variants (SNV) and insertions/deletions (INDEL)
-    - See the [subworkflow doc](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc_strelka2_subworkflow.md) for more information
+    - See the [subworkflow doc](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc_strelka2_subworkflow.md) for more information
```
Comment (dmiller15) [inline workflow/kfdrc-somatic-variant-workflow.cwl:65]: Trailing whitespace.

**Review by dmiller15 — APPROVED** [review]
Mostly formatting stuff worth double checking.

File: workflow/kfdrc-somatic-variant-workflow.cwl (hunk 2 above)
Comment (migbro) [reply workflow/kfdrc-somatic-variant-workflow.cwl:48]: Yeah, the info is a bit sparse, but you do get gene names that exist in this "plasmid" of sorts if it finds it.

**Review by bmennis — APPROVED** [review]
Looks good to me, interesting tool

**Pull request merged**